packages/toolpad-app/src/toolpad/AppEditor/PageEditor/PageOptionsPanel.tsx

```diff
@@ -21,14 +29,14 @@ export default function PageOptionsPanel() {
   const page = appDom.getNode(dom, pageNodeId, 'page');
 
   const handleDisplayModeChange = React.useCallback(
-    (event: React.ChangeEvent<HTMLInputElement>) => {
+    (event: React.MouseEvent<HTMLElement>, newValue: string) => {
```

Comment:
```suggestion
    (event: React.MouseEvent<HTMLElement>, newValue: appDom.PageDisplayMode) => {
```

Just noticed I think we don't need to use `as`, we can just do this.